Dockerfile

```diff
@@ -4,6 +4,10 @@ LABEL maintainer="Star Lab <info@starlab.io>"
 
 RUN mkdir /source
 
+RUN find /etc/yum.repos.d/ -type f -exec sed -i 's/mirrorlist=/#mirrorlist=/g' {} +
```

Comment:
The `RUN` is going to expect to have a single line -- as shown below. You'll need to make sure you string these commands together with `&&` and escape lines with `\` like you would expect.